src/software/backend/simulation/simulator_robot_singleton.h

```diff
@@ -0,0 +1,232 @@
+#pragma once
+
+#include <cinttypes>
+#include <optional>
+
+#include "software/backend/simulation/simulator_robot.h"
+extern "C"
+{
+#include "firmware/main/app/world/firmware_robot.h"
+}
+
+// TODO: These are all hardcoded values copied from firmware/main/physics/physics.h
+// and firmware/main/control/control.h
+// They should be replaced with the proper constants once firmware cleanup is done
+#define GEAR_RATIO 0.5143f  // define as speed multiplication from motor to wheel
+#define WHEEL_RADIUS 0.0254f
+#define WHEEL_SLIP_VOLTAGE_LIMIT 4.25f  // Voltage where wheel slips (acceleration cap)
+#define RPM_TO_VOLT (1.0f / 374.0f)     // motor RPM to back EMF
+#define PHASE_RESISTANCE 1.6f           // adjust this number as calculated
+#define CURRENT_PER_TORQUE 39.21f       // from motor data sheet (1/25.5 mNm)
+#define ROBOT_POINT_MASS 2.48f
+#define ROBOT_RADIUS 0.085f
+#define INERTIAL_FACTOR 0.37f
+#define ROT_MASS (INERTIAL_FACTOR * ROBOT_POINT_MASS)
+#define INERTIA (ROT_MASS * ROBOT_RADIUS * ROBOT_RADIUS)
+#define JERK_LIMIT 40.0f  //(m/s^3)
```

Comment:
We can't include these??

Reply:
Don't know why I didn't think I could before. I included everything from `physics.h`, but couldn't include `JERK_LIMIT` or `WHEEL_MOTOR_PHASE_RESISTANCE` because they depend on IO. I made more specific comments for them and moved the defines to the `.cpp` file